nvim/native.c

```diff
@@ -15,7 +15,7 @@ static int pid_wait(lua_State *L) {
 
 static int pid_wait(lua_State *L) {
   int status;
-  pid_t pid = luaL_checkint(L, 1);
+  pid_t pid = (int)luaL_checkinteger(L, 1);
```

Comment:
Why not cast to `pid_t` instead of assuming it's the same as `int`?

Reply:
@jamessan 
Amended, thanks.